lib/Util.ts

```diff
@@ -154,10 +154,6 @@ export class Util {
       const body1 = body.pipe(new PassThrough());
       const body2 = body.pipe(new PassThrough());
 
-      // Remove unneeded headers
-      response.headers.delete('content-length');
-      response.headers.delete('content-encoding');
```

Comment:
These actually need to be removed, otherwise they can cause issues with binary files (https://github.com/rubensworks/rdf-test-suite.js/commit/1dfee4365254550ece190abb0075b1f5f80b167d).
Can we instead copy the headers object, and omit these entries?